bob-common/src/error.rs

```diff
@@ -156,23 +156,24 @@ impl From<Status> for Error {
         let name = words.next();
         let length = status.message().len();
         match name {
-            None => None,
+            None => Self::failed(format!("Can't parse status from {}", status.message())),
```

Comment:
It would be good to add `status.code()` into the message too

Reply:
Please, use `Debug` formatter for status code, because in `Display` it returns text description, that we don't need. It is better to see just enum value name